examples/java/scenarios/src/main/java/main/Airdrops.java

```diff
@@ -0,0 +1,48 @@
+package main;
+
+import com.google.protobuf.Struct;
+import iog.psg.client.nativeassets.NativeAssetsBuilder;
+import iog.psg.service.nativeassets.AirDrop;
+import iog.psg.service.nativeassets.NativeAssetId;
+import iog.psg.service.nativeassets.TimeBounds;
+
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.IntStream;
+
+public class Airdrops {
+    public static void main(String[] args) throws ExecutionException, InterruptedException {
+        //client setup
+        var client = NativeAssetsBuilder
+                .create("key", "clientId")
```

Comment:
General comment: don't we want to provide a default key,clientId values for a test account which will work out of the box in our production testnet?
WDYT?

Reply:
this may leas to DoS attack